cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -193,6 +196,9 @@ func (s *CMServer) Run(_ []string) error {
 	controllerManager := replicationControllerPkg.NewReplicationManager(kubeClient, replicationControllerPkg.BurstReplicas)
 	go controllerManager.Run(s.ConcurrentRCSyncs, util.NeverStop)
 
+	daemonManager := daemon.NewDaemonManager(kubeClient)
```

Comment:
I think we need to add this to contrib/mesos/pkg/controllermanager/controllermanager.go as well - I forgot about this.